src/components/Postiterie.vue

```diff
@@ -73,11 +83,12 @@ export default {
 
     /*
       FUNCTION previewImage
-      preview the uploaded image
+      preview the uploaded image (and activate the button to track)
       IN : fieldName : the fieldName ;)
            file : the image
     */
     previewImage: function(fieldName, file) {
+            document.getElementById("startTracking").classList.remove("hidden");
```

Comment:
DOM direct manipulation is strongly discouraged if you're using a front-end framework like Vue, Angular or React.